spec/api/comment_thread_spec.rb

```diff
@@ -668,7 +667,7 @@ def test_unicode_data(text)
       let :default_params do
         {body: "new comment", course_id: "1", user_id: User.first.id}
       end
-      it "create a comment to the comment thread" do
+      it "create a comment to the comment thread and marks thread as read for user" do
```

Comment:
I agree that what you describe might take another story. I'm not certain what the solution should be, but you clearly know more about this. If there is still a strong reason to optimize particular calls, it should just be made more clear.